brainscore/submission/evaluation.py

```diff
@@ -79,21 +94,25 @@ def run_evaluation(config_file, work_dir, jenkins_id, db_secret, models=None,
                         ceiled = score.sel(aggregation='center').item(0)
                         error = score.sel(aggregation='error').item(0)
                     finished = datetime.datetime.now()
+                    layer_commitment = str(
+                        model.layer_model.region_layer_map) if submission.model_type == 'BaseModel' else ''
                     result = {
                         'Model': model_id,
                         'Benchmark': benchmark,
                         'raw_result': raw,
                         'ceiled_result': ceiled,
                         'error': error,
                         'finished_time': finished,
-                        'layer' : str(model.layer_model.region_layer_map)
+                        'layer': layer_commitment
```

Comment:
I wrote this in the other PR, but I think this should be something like `'comment': "layers: {layer_commitment}"`